src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2514,6 +2519,57 @@ namespace winrt::TerminalApp::implementation
         return resultPane;
     }
 
+    // Method Description:
+    // - Sets background image and applies its settings (stretch, opacity and alignment)
+    // - Checks path validity
+    // Arguments:
+    // - newAppearance
+    // Return Value:
+    // - <none>
+    void TerminalPage::_SetBackgroundImage(const winrt::Microsoft::Terminal::Control::IControlAppearance& newAppearance)
+    {
+        if (newAppearance.BackgroundImage().empty())
```

Comment:
note: this is quite nearly copy-pasted from `TermControl::_SetBackgroundImage`, so we know that's already good. We must have already evaluated env vars for the settings, so don't need to worry about that here.